util/gen_log_guide.ts

```diff
@@ -0,0 +1,503 @@
+import path from 'path';
+
+import markdownMagic from 'markdown-magic';
+
+import { isLang, languages, NonEnLang } from '../resources/languages';
+import logDefinitions, { LogDefinitionTypes } from '../resources/netlog_defs';
+import NetRegexes from '../resources/netregexes';
+import Regexes from '../resources/regexes';
+import LogRepository from '../ui/raidboss/emulator/data/network_log_converter/LogRepository';
+import ParseLine from '../ui/raidboss/emulator/data/network_log_converter/ParseLine';
+
+type LangObject<T> = { en: T } & { [lang in NonEnLang]?: T };
```

Comment:
I think this is a `LocaleObject<T>` from `trigger.d.ts`?

Reply:
Thank you. I knew we had a type for this already but I kept searching for Lang, instead of Locale.